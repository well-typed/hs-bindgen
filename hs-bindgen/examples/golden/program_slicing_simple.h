#include <stdint.h>

struct foo {
  uint64_t sixty_four;
  uint32_t thirty_two;
};
