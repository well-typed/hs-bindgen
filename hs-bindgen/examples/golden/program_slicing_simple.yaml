version:
  hs_bindgen: 0.1.0
  binding_specification: '1.0'
types:
- headers: program_slicing_simple.h
  cname: uint64_t
  module: Example
  identifier: Uint64_t
  instances:
  - Eq
  - Ord
  - Enum
  - Ix
  - Bounded
  - Read
  - Show
  - Bits
  - FiniteBits
  - Integral
  - Num
  - Real
  - Storable
